src/net.cpp

```diff
@@ -2377,6 +2406,17 @@ bool CConnman::Start(CScheduler& scheduler, const Options& connOptions)
         }
     }
 
+    m_anchors = ReadAnchors(GetDataDir() / ANCHORS_DATABASE_FILENAME);
+    Assert(MAX_BLOCK_RELAY_ONLY_ANCHORS >= 0);
```

Comment:
Why add a runtime assert on the value of a `constexpr` which is known at compilation time? Just add a static_assert if you need to do this (but I don't think this is required at all).

Reply:
Thanks. [Updated](https://github.com/bitcoin/bitcoin/pull/17428#issuecomment-693330527).